### PR TITLE
Fix a couple of typos

### PR DESCRIPTION
**Pull request by knowledgejunkie**
### Summary
Fix typos detected automatically when packaging version 5.07 for Debian

### Motivation
Improve documentation

### References
n/a

### CODE REVIEW

File: lib/JSON/Validator/Schema.pm [hunk 1]
```diff
@@ -867,7 +867,7 @@ JSON::Validator::Schema - Base class for JSON::Validator schemas
   my $store = JSON::Validator::Store->new;
   my $schema = JSON::Validator::Schema::Draft7->new(store => $store);
 
-  # Will not fetch the fike from web, if the $store has already retrived
+  # Will not fetch the fike from web, if the $store has already retrieved
```
Comment (jhthorsen) [inline lib/JSON/Validator/Schema.pm:870]: Maybe fix "fike" as well 😄

**Review by jhthorsen — CHANGES_REQUESTED** [review]

File: lib/JSON/Validator/Schema.pm (hunk 1 above)
Comment (knowledgejunkie) [reply lib/JSON/Validator/Schema.pm:870]: Will do. I'm sure that can be added to lintian.

**Comment by knowledgejunkie** [comment]
Have now read all of the POD in this particular file. Added some more fixes and couple of suggestions.

**Review by jhthorsen — APPROVED** [review]
Thank you!

**Comment by jhthorsen** [comment]
I added some basic spell checking now, so hopefully you don't have to inform me about silly typos in the future: https://github.com/jhthorsen/json-validator/commit/9a43b8c47311dc75548d9f45640834f3b196cea3

I do appreciate the rephrasing of sentences though! 👍

**Pull request merged**